graspologic/cluster/divisive_cluster.py

```diff
@@ -137,20 +149,28 @@ def fit(self, X):
         self.fit_predict(X)
         return self
 
-    def fit_predict(self, X):
+    def fit_predict(self, X, fcluster=False, level=None):
         """
         Fits clustering models to the data as well as resulting clusters
         and using fitted models to predict a hierarchy of labels
 
         Parameters
         ----------
         X : array-like, shape (n_samples, n_features)
+        fcluster: bool, default=False
+            if True, returned labels will be a hierarchy of flat clusterings,
+            one for each level
```

Comment:
just documenting from our conversation wednesday - the wording here confused me in terms of what this is actually doing